src/import-service/serverless.yml

```diff
@@ -0,0 +1,37 @@
+service: ImportService
+
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+  region: eu-central-1
+  httpApi:
+    cors: true
+  iamRoleStatements:
+    - Effect: Allow
+      Action: s3:*
+      Resource:
+        - arn:aws:s3:::cloudx-aws-practitioner-js
+    - Effect: Allow
+      Action: s3:*
+      Resource:
+        - arn:aws:s3:::cloudx-aws-practitioner-js/*
```

Comment:
Do we need these 2 almost similar statements? Not sure maybe you're right. I'm just curious =)